test/onnx/test_onnx_opset.py

```diff
@@ -206,6 +207,21 @@ def forward(self, x):
         ops = {9 : ops, 10 : ops}
         check_onnx_opsets_operator(MyModule(), x, ops, opset_versions=[9, 10], training=False)
 
+    def test_gather_from_index_select(self):
+        class MyModule(Module):
+            def forward(self, x):
+                return torch.index_select(x, dim=1, index=torch.tensor(2))
+
+        x = torch.randn(3, 4)
+
+        ops = [
+            {"op_name" : "Constant"},
+            {"op_name" : "Constant"},
+            {"op_name" : "Reshape"},
+            {"op_name" : "Gather", "attributes" : [{"name" : "axis", "i" : 1, "type" : 2}]}]
+        ops = {9 : ops}
+        check_onnx_opsets_operator(MyModule(), x, ops, opset_versions=[9])
```

Comment:
this tests fail in CI. IMO, opset test is for checking some special structure of the graph. Do we have any particular reason to check the structure of the graph?  I think end to end tests should be good enough in this case.

Reply:
Good point. I removed this test.